package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "awesome-books",
+  "version": "1.0.0",
+  "description": "- [📖 About the Project](#about-project)\r   - [🛠 Built With](#built-with)\r     - [Tech Stack](#tech-stack)\r     - [Key Features](#key-features)\r   - [🚀 Live Demo](#live-demo)\r - [💻 Getting Started](#getting-started)\r   - [Setup](#setup)\r   - [Prerequisites](#prerequisites)\r   - [Deployment](#triangular_flag_on_post-deployment)\r - [👥 Authors](#authors)\r - [🤝 Contributing](#contributing)\r - [⭐️ Show your support](#support)",
+  "main": "script.js",
+  "scripts": {
+    "build": "webpack"
+  },
```

Comment:
- [OPTIONAL] I can see you're using Webpack, so I suggest adding `webpack serve` to be able to run the project using `npm start` command 👍

```json
  "scripts": {
    "build": "webpack",
    "start": "webpack serve --open"
  }
```